models/offer.js

```diff
@@ -15,7 +12,10 @@ const offerSchema = new Schema(
     },
     proficiencyLevel: {
       type: String,
-      enum: SUBJECT_LEVEL_ENUM,
+      enum: {
+        values: SUBJECT_LEVEL_ENUM,
+        message: `Proficiency level can be only one of these: ${SPOKEN_LANG_ENUM.toString()}`
```

Comment:
i guess it is better to say
can be either of these:

Reply:
make it the same way everywhere

Reply:
either is used to express a choice between two options
here i have more than two, so i used one of these

Reply:
wrong

https://ell.stackexchange.com/questions/63428/can-either-be-used-for-more-than-two-items

Reply:
ok, got it